pages/test-evaluate/evaluation-tools/index.pl.md

```diff
@@ -0,0 +1,136 @@
+
+<div class="video-card" id="video-intro">
+  {% include video-player-data.html
+      video-id="tools-for-evaluating"
+      yt-id="bn1XJSjc_qM"
+  %}
+  <p><a href="#alternatives">Transkrypcja filmu „Omówienie narzędzi oceny” </a></p>
+</div>
+
+[Lista narzędzi oceny dostępności cyfrowej](/test-evaluate/tools/list/)
+:   Zawiera informacje na temat ponad 100 narzędzi. Możesz skorzystać z filtrów, aby zawęzić listę do typów narzędzi, które Cię interesują.
+
+[[Wybór narzędzi oceniających dostępność cyfrową]](/test-evaluate/tools/selecting/)
+:   Zawiera wskazówki dotyczące wyboru narzędzi. Opisuje cechy i funkcjonalność różnych typów narzędzi oceny oraz omawia kwestie, które należy wziąć pod uwagę w konkretnej sytuacji.
+
+Powiązane materiały przedstawiono w: [[Omówienie oceniania dostępności internetowej]](/test-evaluate/)
+
+<hr>
+
+{% include excol.html type="start" id="video-intro-transcript" %}
+
+##  Omówienie narzędzi diagnostycznych {#alternatives}
+
+{% include excol.html type="middle" %}
+
+_Ten film jest również dostępny na serwerze W3C: Film: Omówienie narzędzi oceny (format pliku: MP4, rozmiar pliku: 51MB MB)](https://media.w3.org/wai/evaluation-intros/tools-for-evaluating.mp4)._
+
+###  Transkrypcja tekstowa z opisem elementów wizualnych dla filmu Wybór i stosowanie narzędzi oceny {#transcript}
+
+Filmy zawierają podstawową animację ilustrującą to, co mówi głos spoza ekranu. Ludzie są przedstawieni za pomocą symboli graficznych.
+
+<table aria-labelledby="transcript">
+  <thead>
+    <tr>
+      <th width="50%">Dźwięk</th>
+      <th>Obraz</th>
+    </tr>
+  </thead>
+<tbody>
+  <tr>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+  </tr>
+  <tr>
+    <td>Istnieją programy i usługi internetowe, które pomogą Ci rozpoznać problemy z dostępnością.</td>
+    <td>Otwiera się skrzynka narzędziowa przy komputerze. Ze skrzynki wychodzi lupa, aby sprawdzić stronę internetową na komputerze. Lupa powiększa słowo „dostępność”.</td>
+  </tr>
+  <tr>
+    <td>Mogą one zaoszczędzić wiele czasu i wysiłku podczas oceny oraz zapobiec powstawaniu nowych barier dostępności.</td>
+    <td>Oszczędzaj czas i wysiłek. Unikaj barier dostępności.</td>
+  </tr>
+  <tr>
+    <td>Jednak narzędzia nie mogą zrobić wszystkiego. Niektórych testów dostępności po prostu nie można zautomatyzować i muszą być przeprowadzone ręcznie.</td>
+    <td>Osoba obok komputera ze stroną internetową używa lupy ze słowem „dostępność”.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia poprowadzą Cię przez te testy, których nie można zautomatyzować.</td>
+    <td>Lista kontrolna obok komputera.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia sprawdzają tylko jedną stronę na raz, inne mogą sprawdzać całe witryny.</td>
+    <td>Strona w witrynie internetowej jest skanowana. Nad elementami strony pojawia się kolejno znak błędu, powodzenia i znak zapytania.</td>
```

Comment:
@zwiastunsw This description seems to have been reworded. Similar to https://github.com/w3c/wai-website/pull/561#discussion_r1679355627